src/Sulu/Bundle/MediaBundle/Resources/js/package.json

```diff
@@ -4,13 +4,15 @@
     "license": "MIT",
     "repository": "https://github.com/sulu/sulu.git",
     "dependencies": {
-        "sulu-admin-bundle": "file:../../../AdminBundle/Resources/js",
-        "react": "^15.6.0",
         "classnames": "^2.2.5",
-        "mobx-react": "^4.2.1"
+        "mobx-react": "^4.2.1",
+        "react": "^15.6.0",
+        "react-clipboard.js": "^1.1.2",
```

Comment:
This file is missing in `/package.json` of `sulu/sulu`, that's probably also the reason the tests are failing.

Reply:
I have added "sulu-media-bundle": "file:src/Sulu/Bundle/MediaBundle/Resources/js" as a dependency. Should that fix the issue?